src/components/report/VariableControls.js

```diff
@@ -105,6 +129,9 @@ export default {
 				axis,
 				force || this.currentStates[axis] === false
 			);
+			if (axis === "opsz") {
```

Comment:
I don't think this belongs in the engine, as it's site logic. This specific code turns axex on/off in the interface, in response to some user interaction (e.g. pressing on/off button). In the specific case of `opsz`, it's important to let other UI elements know this axis has changed, so they can respond to that. In this specific case, the UI element that controls the font size should stop syncing up with the `opsz` value.

I hope this explanation makes sense. If you still think this should be moved elsewhere, let discuss!

Reply:
That makes sense. What do you think about exposing the axis names from the library, to prevent relying on strings? Something like `import { opsz } from "@wakamai-fondue/engine/axis"` or something like that.